docs/js/dynamics.js

```diff
@@ -0,0 +1,113 @@
+const comedians = [
+  {
+    name: 'MC Kisoli',
+    biography: "Bachelor's Degree, Government Worker, Lecturer",
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.Maiores error blanditiis necessitatibus soluta. Eveniet, quae.',
+    imgSrc: './images/mtubaki5.jpg',
+    imgAlt: 'Mtubaki1',
+  },
+  {
+    name: 'Joe Master',
+    biography: "Bachelor's Degree, Government Worker, Lecturer",
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.Maiores error blanditiis necessitatibus soluta. Eveniet, quae.',
+    imgSrc: './images/mtubaki2.jpg',
+    imgAlt: 'Mtubaki2',
+  },
+  {
+    name: 'Mtoto wa Dada',
+    biography: 'Business Owner',
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.Maiores error blanditiis necessitatibus soluta. Eveniet, quae.',
+    imgSrc: './images/mtubaki4.jpg',
+    imgAlt: 'Mtubaki3',
+  },
+  {
+    name: 'Dogo Pepe',
+    biography: "Bachelor's Degree, Private Sector Worker",
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.Maiores error blanditiis necessitatibus soluta. Eveniet, quae.',
+    imgSrc: './images/mtubaki1.jpg',
+    imgAlt: 'Mtubaki4',
+  },
+  {
+    name: 'Kipatoshi',
+    biography: "Bachelor's Degree, Government Worker, Lecturer",
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.Maiores error blanditiis necessitatibus soluta. Eveniet, quae.',
+    imgSrc: './images/mtubaki3.jpg',
+    imgAlt: 'Mtubaki5',
+  },
+];
+
+const comediansCards = document.querySelector('#comedians-cards');
+let comediansNumbertoShow = 2;
+let nextStartIdx = 0;
+
+// Takes number of comedians to show, and where
+// to start in list and adds card ?
+// markup for each comedian in the list
```

Comment:
- [OPTIONAL ] Please do not leave unused code as a comment in your codebase. unused code as comments makes your code not clean. Kindly remove all unused code this will make your code mush noise, It is always a good practice to keep your code clean from such comments.

Reply:
Yes, this is not an unused code. It is a comment a way to document my code. If I remove it, would the code be better from a maintainability point of view?